ee/clickhouse/queries/funnels/funnel_time_to_convert.py

```diff
@@ -108,7 +108,7 @@ def get_query(self) -> str:
             RIGHT OUTER JOIN (
                 /* Making sure bin_count bins are returned */
                 /* Those not present in the results query due to lack of data simply get person_count 0 */
-                SELECT histogram_from_seconds + number * bin_width_seconds AS bin_from_seconds FROM system.numbers LIMIT {bin_count_identifier} + 1
+                SELECT histogram_from_seconds + number * bin_width_seconds AS bin_from_seconds FROM system.numbers LIMIT ifNull({bin_count_identifier}, 0) + 1
```

Comment:
There should be a @snapshot_clickhouse_queries code path covering this - really surprised this doesn't have coverage.

Do we want to add the test here or e.g. tag @neilkakkar to take a look?

Reply:
Tests added. Let me know if it's 👍